### PR TITLE
Added docker build and push steps

### PR DESCRIPTION
**Pull request by vinod-1**
Added steps to docker build and push.
Created a workflow for deploy which will execute on test ruby success.
Remaining to create a shell script in server

### CODE REVIEW

**Comment by vinod-1** [comment]
Screenshot_2019-10-24_22-21-40

File: .circleci/config.yml [hunk 1]
```diff
@@ -176,6 +176,18 @@ jobs:
...
+    <<: *defaults
+    steps:
+      - setup_remote_docker: 
+      - *attach_workspace
+      - run:
+          name: Login to DockerHub
+          command: docker login -u $DOCKER_USER -p $DOCKER_PASS
+      - run: docker build -t vinodplank/masto:$CIRCLE_BUILD_NUM .
+      - run: docker push vinodplank/masto:$CIRCLE_BUILD_NUM
+      - run: ssh -o StrictHostKeyChecking=no ubuntu@13.235.249.241 "sed -i 's/masto:[0-9][0-9]*/masto:$CIRCLE_BUILD_NUM/g' ~/CircleCI/juna-apps.yml"
```
Comment (plank-tools) [inline .circleci/config.yml]: The ip address should be ready from environment variables. Also change ip address to internal hostname.

Comment (vinod-1) [reply .circleci/config.yml]: Changed hostname and docker username to environment variables

File: .circleci/config.yml [hunk 2]
```diff
@@ -176,6 +176,18 @@ jobs:
...
       - run: bundle exec i18n-tasks check-consistent-interpolations
+  publish:
+    <<: *defaults
+    steps:
+      - setup_remote_docker: 
+      - *attach_workspace
+      - run:
+          name: Login to DockerHub
+          command: docker login -u $DOCKER_USER -p $DOCKER_PASS
+      - run: docker build -t vinodplank/masto:$CIRCLE_BUILD_NUM .
```
[the remark this answers is not in the document]
Comment (vinod-1) [reply .circleci/config.yml]: Made this to environment variable

**Pull request merged**